### ISSUE TITLE
React-toastr throwing error

### ISSUE THREAD

**Issue by tarunthota3** [tomchentw/react-toastr#112]
I have been receiving a new error after the latest release 2.9.2, before it was working fine.
The error which i got is - "import is not defined"

**Comment by culshaw** [comment]
I get something similar.

When using 2.9.2 with babelify/browserify
```
[16:45:23] Browserify Error 
~/node_modules/react-toastr/lib/index.js:1
import { default as ToastContainer } from "./ToastContainer";
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'
```

**Comment by tomchentw** [comment]
Released [v2.9.3](https://github.com/tomchentw/react-toastr/releases/tag/v2.9.3)

**Comment by tarunthota3** [comment]
Thanks for resolving the issue.. :)
